targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/serial_api.c

```diff
@@ -166,12 +167,12 @@ typedef enum {
 /**
  * UARTE state. One for each instance.
  */
-static nordic_uart_state_t nordic_nrf5_uart_state[NRFX_UART_ENABLED_COUNT] = { 0 };
+static nordic_uart_state_t nordic_nrf5_uart_state[NRFX_UARTE_ENABLED_COUNT] = { 0 };
 
 /**
  * Array with UARTE register pointers for easy access.
  */
-static NRF_UARTE_Type *nordic_nrf5_uart_register[NRFX_UART_ENABLED_COUNT] = {
+static NRF_UARTE_Type *nordic_nrf5_uart_register[NRFX_UARTE_ENABLED_COUNT] = {
     NRF_UARTE0,
 #if UART1_ENABLED
```

Comment:
I guess this should be defined?

Reply:
Thanks for reviewing this, you are correct.
`UART1_ENABLED` shall be defiend as 1, somehow the updating Nordic SDK to 15 disabled it.